Svc/ActiveLogger/test/ut/output/ActiveLoggerImpl.cpp.gcov

```diff
@@ -245,7 +245,7 @@
        20:  243:                fileWriteSize = sizeof(delimiter);
        20:  244:                stat = file.write(&delimiter,fileWriteSize);
        20:  245:                if (stat != Os::File::OP_OK) {
-        1:  246:                    this->log_WARNING_HI_ALOG_FILE_WRITE_ERR(LOG_WRITE_FATAL_DELIMETER,stat);
+        1:  246:                    this->log_WARNING_HI_ALOG_FILE_WRITE_ERR(LOG_WRITE_FATAL_DELIMITER,stat);
```

Comment:
Not sure about changing `.gcov`

Reply:
This is a report file not used elsewhere.